### ISSUE TITLE
Strange multiword with posar as lemma of sobre

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Catalan-AnCora#2]
```
53-54   posar-lo        _       _       _       _       _       _       _       _
53      posar   posar   VERB    VERB    VerbForm=Inf    48      ccomp   _       _
54      lo      ell     PRON    PRON    Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs  53      obj     _       _
55      sobre   posar   ADP     ADP     AdpType=Prep    53      aux     _       MWE=sobre_avís|MWEPOS=AUX
56      avís    sobre   NOUN    NOUN    Number=Plur     55      compound        _       _
```

**Comment by hectormartinez** [comment]
Looks like an offset bug when lemmatizing parts of former MWEs that
were_tokenized_together.

Have you observed any other cases like that?

2017-07-25 9:12 GMT+02:00 Francis Tyers <notifications@github.com>:

**Comment by ftyers** [comment]
Yeah, there are a few more:

```
17      se'n    anar-se'n       VERB    VERB    Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin    0       root    _       MWE=se'n_anirà_a_fer_punyetes|MWEPOS=VERB
18      anirà   a       VERB    VERB    Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin    17      compound        _       _
19      a       fer     ADP     ADP     AdpType=Prep    17      compound        _       _
20      fer     punyetes        VERB    VERB    Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin    17      compound        _       _
21      punyetes        _       VERB    VERB    Mood=Ind|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin    17      compound        _       _
```

```
18-19   dur-hi  _       _       _       _       _       _       _       _
18      dur     dur     VERB    VERB    VerbForm=Inf    14      acl     _       _
19      hi      hi      PRON    PRON    Person=3|PronType=Prs   18      obl     _       _
20      a       dur     ADP     ADP     AdpType=Prep    18      aux     _       MWE=a_terme|MWEPOS=AUX
21      terme   a       NOUN    NOUN    Number=Sing     20      compound        _       _
```

I found them when working on tokenisation, I can probably come up with a reasonably complete list if you like?

**Comment by hectormartinez** [comment]
That would be really helpful -- how do we coordinate the change,  @
<https://github.com/danzeman>danzeman <https://github.com/danzeman> ?

2017-07-25 13:17 GMT+02:00 Francis Tyers <notifications@github.com>:

**Comment by ftyers** [comment]
Here is the list. There are some things which won't be errors, but there will be some that are, there are only 54, so it shouldn't be too hard to go through and check manually. Basically I looked for lemmas that didn't start with the same character as the surface form and removed some obvious stuff like 'vaig--anar' 'fou--ser' etc.

```
avís	sobre
càrrec	_
caure	_
ciutat	_
col·lector	lector
curar	-B2
deixa	caure
em	jo
enlairar	_
enrere	_
ens	jo
es	deixar-se
es	fer-se
farà	enrere
fer	punyetes
fet	-A2
mi	jo
m'	jo
ne	en
n'	en
nº3	3
nº5	5
nº7	7
nº	N
nosaltres	jo
nos	es
nos	jo
per	_
portar	_
punyetes	_
s'adiuen	adir-se
sa	es
se	ell
se	es
s'	ell
se'n	anar-se'n
s'	es
se	's
se	-se
ses	es
si	ell
sigut	estar
sobre	posar
terme	a
usos	ús
us	tu
veure	a
veure	res
vint	20
vosaltres	tu
vostès	tu
vostè	tu
```

**Comment by dan-zeman** [comment]
@hectormartinez : I don't remember what is the last status of our working pipeline, i.e. whether we still can and want to maintain the conversion procedure from the AnCora annotation. Probably we have not broken it yet. (For @ftyers : My scripts convert AnCora to UD. They are not perfect at using all the information from the original, so there is hope that in the future we can improve and re-run them. Things done by Héctor are all scripted as well and can be run on UD input, i.e. they can be re-applied on improved conversion. But you cannot fix anything manually.)

Now I wonder whether maintaining this workflow is worth the limited flexibility. Even if we abandon it, we will not lose the original data completely. We can still try to map it on UD if we think there is something useful we lost. But the mapping may not be easy of course.

This particular bug seems to have something to do with Héctor's software, so maybe fixing and re-running the software is the way to go now. But in the future we will have to think about the workflow anyway.

**Comment by cayorodriguez** [comment]
Hi, at the Barcelona Supercomputing Center we are working on extending and creating catalan datasets, including UD treebanks. Maybe we can become full contributors. We have catalan linguists and developers. Where can we find the documentation of what's being done with the catalan Ancora for UDs, and who can we coordinate this with? Thanks

**Comment by jnivre** [comment]
This sounds great. Thanks for joining the crowd. As far as I know, there is no ongoing work on the Catalan Ancora treebank, but maybe someone else knows better. In any case, the contact persons for the treebank should be available in the README file.

**Comment by dan-zeman** [comment]
> I don't remember what is the last status of our working pipeline, i.e. whether we still can and want to maintain the conversion procedure from the AnCora annotation.

I declare this dataset detached from my conversion procedure from the CoNLL 2009 version. Independent maintenance of the UD version seems now to be more advantageous than keeping the data in sync with the old version.

I have fixed the lemma mismatches caused by the offset bug in MWE decomposition; that was the main topic of this issue, which can now be closed. I also fixed some other errors (e.g., non-projective punctuation and decomposition of preposition-article contractions).

Other fixes and contributions are of course welcome.
